content/200-orm/050-overview/500-databases/200-database-drivers.mdx

```diff
@@ -26,7 +26,7 @@ There are 2 different types of driver adapters:
 
 ### Database driver adapters
 
-You can connect to your database using a Node.js-based driver from Prisma Client using a database driver adapter. Prisma maintains the following database driver adapters:
+You can connect to your database using a Node.js-based driver from Prisma Client using a database driver adapter. Prisma Client maintains the following database driver adapters:
```

Comment:
The "Prisma" here refers to "Prisma, the company" - us.
If we need to change that, I would suggest to go with "Prisma ORM" anyway as that is less weird.